File: c/square.c
#include <stdio.h>

int main()
{
    int myNum, myNum2;
    printf("Enter a number: ");
    scanf("%d", &myNum);
    for (int i = 1; i <= myNum; i++) {
        printf("* ");
    if(i == myNum) {
        printf("\n");
    }}
    myNum2 = myNum - 2;
    for (int i = 1; i <= myNum2; i++){
        printf("* ");
        for (int j = 1; j <= myNum2; j++){
            printf("  ");
        }
        printf("* \n");
    }
    for (int i = 1; i <= myNum; i++) {
        printf("* ");
    if(i == myNum) {
        printf("\n");
    }}    
}

File: python/square.py
def square(n):
    print(n * "* ")
    m = n - 2
    for i in range(m):
        print("* " + m * "  " + "* ")
    print(n * "* ")

n = input("number : ")
n = int(n)

if n >= 2:
    square(n)
else:
    print("Wrong! enter n >= 2 number.")
